up42/asset.py

```diff
@@ -103,14 +103,24 @@ def update_metadata(
         Update the metadata of the asset.
 
         Args:
-            title: The title string to be assigned to the asset.
-            tags: A list of tag strings to be assigned to the asset.
+            title: The title string to be assigned to the asset. No value will keep the existing title.
+            tags: A list of tag strings to be assigned to the asset. No value will keep the existing tags.
 
         Returns:
             The updated asset metadata information
         """
         url = endpoint(f"/v2/assets/{self.asset_id}/metadata")
-        body_update = {"title": title, "tags": tags, **kwargs}
+        if kwargs:
```

Comment:
if say a user intends to actually remove the title and tags, can he still do it by passing an empty string or null?

Reply:
Nop, it will be ignored. That is the tradeoff of the solution. We should add an argument to the signature to remove titles. That is the proposal to solve the issue. Maybe there is a better way :)

Reply:
we need to change the default value here from None to something like [this](https://github.com/Python-Optional/optional.py/blob/main/optional/nothing.py). Please add it to the utils with the corresponding tests. This way we will be able to separate the case when None value is provided versus the case when it's not provided at all

Reply:
I have updated following a simpler approach but with the same results. Please check it and let me know what you think. Also, I have update the tests.